client/src/components/NewReviewForm.js

```diff
@@ -0,0 +1,116 @@
+import React, {useState } from "react"
+import ErrorList from "./layout/ErrorList"
+
+const NewReviewForm = ({ pizzaId, pizzaPlace, setPizzaPlace, currentUser }) => {
+  const [newReview, setNewReview] = useState({
+    userId: currentUser.id,
```

Comment:
the form should not be responsible for keeping track of the currentUser. The back-end Express application is managing that for us through the session. The front-end form should only track the info provided by the user. When the data is submitted to the back-end, we can access the currently signed in user and attach it to the new Review before we persist it to the database